icicle/src/hash/merkle_c_api.cpp

```diff
@@ -119,14 +119,16 @@ const std::byte* icicle_merkle_tree_get_root(icicle::MerkleTree* tree, size_t* o
 eIcicleError icicle_merkle_tree_get_proof(
   icicle::MerkleTree* tree,
   const std::byte* leaves,
+  uint64_t leaves_size,
```

Comment:
this is breaking rust right?

Reply:
right.
get proof must recieve leaves_size for padding